dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -35,6 +35,16 @@ RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return
 {%- endfor %}
 {%- endif %}
 
+{%- if CONFIGURED_ARCH != "amd64" %}
```

Comment:
>CONFIGURED_ARCH != "amd64" [](start = 7, length = 26)

Could you positively test for arm ARCH only? What is the nature of this issue you met? Please add some code comments.

Reply:
Done